tests/TestTrust.py

```diff
@@ -52,52 +53,70 @@ def test_signFileAndVerify(self, init_trust):
         filepath_signed = os.path.join(temp_dir, _folder_names[0], _subfolder_names[0], _file_names[0])
         filepath_unsigned = os.path.join(temp_dir, _folder_names[1], _subfolder_names[0], _file_names[2])
 
+        # Attempt to sign a file.
         assert signFile(private_path, filepath_signed, _passphrase)
 
+        # Check if we're able to verify the file we just signed.
         assert trust_instance.signedFileCheck(filepath_signed)
+        assert violation_callback.call_count == 0  # No violation
+
+        # Check if the file we didn't sign notifies us about this.
         assert not trust_instance.signedFileCheck(filepath_unsigned)
+        assert violation_callback.call_count == 1
+
+        # An unknown file is also seen as an invalid one.
         assert not trust_instance.signedFileCheck("file-not-found-check")
+        assert violation_callback.call_count == 2
 
+        # The signing should fail if we disable the key (since we can't confirm anything)
         public_key = copy.copy(trust_instance._public_key)
         trust_instance._public_key = None
         assert not trust_instance.signedFileCheck(filepath_signed)
-        assert violation_callback.call_count > 0
+        assert violation_callback.call_count == 3
         violation_callback.reset_mock()
         trust_instance._public_key = public_key
 
+        # Oh noes! Someone changed the file!
         with open(filepath_signed, "w") as file:
             file.write("\nPay 10 Golden Talents To Get Your Data Back Or Else\n")
```

Comment:
What are golden talents and where do I get them? Need those nature pics restored ASAP

Reply:
https://en.wikipedia.org/wiki/Talent_(measurement)

Reply:
https://en.wikipedia.org/wiki/Talent_(measurement)